docs/crocks/List.md

```diff
@@ -0,0 +1,111 @@
+# List
+
+`List a`
+
+--
+
+--
+
+```js
+--
+```
+
+`List` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`empty`](#empty), [`fromArray`](#fromArray), [`of`](#of) | [`ap`](#ap), [`chain`](#chain), [`concat`](#concat), [`cons`](#cons), [`empty`](#empty), [`equals`](#equals), [`filter`](#filter), [`head`](#head), [`map`](#map), [`of`](#of), [`reduce`](#reduce), [`reject`](#reject), [`sequence`](#sequence), [`tail`](#tail), [`toArray`](#toArray), [`traverse`](#traverse), [`value`](#value) |
+
+## Constructors
+
+### empty
+
+`List m => () -> m ()`
```

Comment:
Even though the List is empty, it still needs to align with a type. This should be:
`List m => () -> m a`